marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.tsx

```diff
@@ -139,7 +138,7 @@ const SelectedFacets: React.FC<Props> = (props) => {
                 data-cy={`clear-${item.rangeValues.lowerBound}`}
               >
                 <Icon type='close'/>
-                {facetName + ': ' + item.rangeValues.lowerBound + ' ~ ' + item.rangeValues.upperBound}
```

Comment:
@saarimrahman any reason why facetName is removed ? As per the requirement it should be facetName:values right ?

Reply:
See Yash's comment here: https://project.marklogic.com/jira/browse/DHFPROD-5412?focusedCommentId=181039&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-181039

Reply:
@saarimrahman I think you misunderstood the requirement, Yash comment was to not show the facetName which is same as entityName